_data/sidebars/marketplace_dev_sidebar.yml

```diff
@@ -506,6 +506,9 @@ entries:
                 include_versions:
                 - "202108.0"
 
+              - title: "Glue API: Marketplace Shopping Lists feature integration"
```

Comment:
this will be added after the release. Now we don't need to add this guide to the sidebar 
```suggestion

```